src/pycram/datastructures/world.py

```diff
@@ -351,6 +351,13 @@ def get_object_by_id(self, obj_id: int) -> Object:
         """
         return list(filter(lambda obj: obj.id == obj_id, self.objects))[0]
 
+    def get_scene_objects(self) -> List[Object]:
+        """
+        Return all objects in the world except the robot, floor and apartment.
+        :return: A list of all objects in the world except the robot, floor and apartment.
+        """
+        return [obj for obj in self.objects if obj not in {self.robot, Floor, Apartment}]
```

Comment:
Floor and Apartment are PyCRAP types, filtering this way will not work as intended. You need to find the respective object by their type first. 
Also, please add a new line between the doc string and the :return:

Reply:
Furthermore, if you comment just describes the return value you can just put it behind the :return: so nothing getsduplicated